source/includes/_request-dictionary.md.erb

```diff
@@ -376,7 +376,7 @@ Name | Description
 </p>
 
 <p>
-  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddargetter-how">
+  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddar-how">
```

Comment:
@mattbatman Need to make this and others conform to current KB article permalinks in Tender.